[DE] Add `Music theory/Syncopation` (#10176)
ppy/osu-wiki: wiki/Music_theory/Syncopation/de.md

---
stub: true
---

# Synkope

> Eine **Synkope** ist die regelmäßige Akzentuierung oder Betonung von Noten an ungewöhnlichen Stellen. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks gibt einen gleichmäßigen [Rhythmus](/wiki/Music_theory/Rhythm) aus starken und schwachen Beats vor, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen versetzt werden. Beim [Beatmapping](/wiki/Beatmapping) sind Synkopen normalerweise zu erkennen, wenn [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, wie zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste.

Reviewer: Should we distinguish `(eine) Synkope` and `Synkope/Synkopieren`? According to the German sources, `eine Synkope`, which is what you're using here, only describes one instance of this irregular rhythm construct. The english osu-wiki article refers to `Syncopation` (not `a syncopation` (singular)), which means the process of using syncopations regularly throughout the song. To be technically correct here, I think we'd need to use `Synkopieren` (a `substantiviertes Verb`), which doesn't sound great though. Something like this maybe:

```suggestion
**Synkopieren** beschreibt die regelmäßige rhythmische Betonung von Noten, die üblicherweise nicht betont werden. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks gibt einen gleichmäßigen [Rhythmus](/wiki/Music_theory/Rhythm) aus starken und schwachen Beats vor, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen versetzt werden. Beim [Beatmapping](/wiki/Beatmapping) sind Synkopen normalerweise zu erkennen, wenn [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, wie zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste.
```

> Eine **Synkope** ist die regelmäßige Akzentuierung oder Betonung von Noten an ungewöhnlichen Stellen. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks gibt einen gleichmäßigen [Rhythmus](/wiki/Music_theory/Rhythm) aus starken und schwachen Beats vor, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen versetzt werden. Beim [Beatmapping](/wiki/Beatmapping) sind Synkopen normalerweise zu erkennen, wenn [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, wie zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste.

Reviewer: ```suggestion
Eine **Synkope** ist die regelmäßige Akzentuierung oder Betonung von Noten an ungewöhnlichen Stellen. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks deutet ein [rhythmisches](/wiki/Music_theory/Rhythm) Muster aus starken und schwachen Beats an, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem Akzente in den Offbeat oder auf unbetonte Stellen gesetzt werden. Beim [Beatmapping](/wiki/Beatmapping) sind Synkopen normalerweise zu erkennen, wenn [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, wie zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste.
```

> Eine **Synkope** ist die regelmäßige Akzentuierung oder Betonung von Noten an ungewöhnlichen Stellen. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks gibt einen gleichmäßigen [Rhythmus](/wiki/Music_theory/Rhythm) aus starken und schwachen Beats vor, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen versetzt werden. Beim [Beatmapping](/wiki/Beatmapping) sind Synkopen normalerweise zu erkennen, wenn [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, wie zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste.

Reviewer: ```suggestion
Eine **Synkope** ist die regelmäßige Akzentuierung oder Betonung von Noten an ungewöhnlichen Stellen. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks gibt einen gleichmäßigen [Rhythmus](/wiki/Music_theory/Rhythm) aus starken und schwachen Beats vor, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen versetzt werden. Beim [Beatmapping](/wiki/Beatmapping) kann man Synkopen normalerweise daran erkennen, dass [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, also zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste platziert sind.
```

> **Synkopen** sind die Akzentuierung von Noten, die üblicherweise nicht betont werden. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks deutet ein [rhythmisches](/wiki/Music_theory/Rhythm) Muster aus starken und schwachen Beats an, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen gesetzt werden. Beim [Beatmapping](/wiki/Beatmapping) kann man Synkopen normalerweise daran erkennen, dass [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, also zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste platziert sind.

Reviewer: ```suggestion
**Synkopen** sind Akzentuierungen von Noten, die üblicherweise nicht betont werden. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks deutet ein [rhythmisches](/wiki/Music_theory/Rhythm) Muster aus starken und schwachen Beats an, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen gesetzt werden. Beim [Beatmapping](/wiki/Beatmapping) kann man Synkopen normalerweise daran erkennen, dass [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, also zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste platziert sind.
```

> Das [Insane-Level von *Chasers - Lost* erstellt von fanzhen0019](https://osu.ppy.sh/beatmapsets/151114#osu/372628) enthält als einfaches Beispiel [Slider](/wiki/Gameplay/Hit_object/Slider)-Patterns mit [Sliderköpfen](/wiki/Gameplay/Hit_object/Slider/Sliderhead), die sowohl im Onbeat als auch im Offbeat liegen.

Reviewer: I think we should avoid `Onbeat`:

```suggestion
Das [Insane-Level von *Chasers - Lost* erstellt von fanzhen0019](https://osu.ppy.sh/beatmapsets/151114#osu/372628) enthält als einfaches Beispiel [Slider](/wiki/Gameplay/Hit_object/Slider)-Patterns mit [Sliderköpfen](/wiki/Gameplay/Hit_object/Slider/Sliderhead), die sowohl auf dem Beat als auch auf dem Offbeat liegen.
```

Author: My intention was to avoid `Onbeat` as well.

Einige Slider-Patterns, wie zum Beispiel das allererste, folgen der Synth-Melodie, die mit den Trommelschlägen synkopiert ist, und landen auf roten Strichen auf der Zeitleiste:

> ![Beispiel eines Sliderskopfes, der einem synkopischen Rhythmus folgt](img/syncopated-sliders.png)

Reviewer: ```suggestion
![Beispiel eines Sliderkopfes, der einem synkopischen Rhythmus folgt](img/syncopated-sliders.png)
```
